src/DiscordServer.ts

```diff
@@ -244,7 +244,7 @@ export class DiscordServer {
         }
 
         if (data.length > 0 && data != null) {
-          var textMatcher = new RegExp(/(?<=\[Text]\n)(.*)(?:=\n\[Options|\n\n)/ms)
```

Comment:
This old regex would fail on the example use-case unless there were two double-lines at the end:
```
[Text]
You meow at the cat. It stares deep into your soul, implying a deep understanding.

You feel the cat could write volumes on what it means to be alive, filled with rich and deep introspective that - 

`the cat knocks over your coffee and meows`
```
It would also incorrectly pickup `[Options]` if there were two double lines at the end.